entrypoint.sh

```diff
@@ -5,6 +5,9 @@ if test -n "$S3_BUCKET" ; then
 var s3_bucketName = '${S3_BUCKET}';
 var s3_access_key = '${S3_ACCESS_KEY}';
 var s3_secret_key = '${S3_SECRET_KEY}';
+var s3_host = '${S3_HOST}'; // S3 host, when self hosting s3
+var s3_bucketPathPrefix = '${S3_BUCKET_PATH_PREFIX}'; // S3 bucket path prefix
+var s3_ForcePathStyle = ${S3_FORCE_PATH_STYLE}; // true or false
```

Comment:
Does it still work when the environment variables are empty? That will assign empty strings to the js variables now.

Reply:
i read on StackOverflow and ask ChatGPT what happens if they are empty or undefined. it said it was okay, and the options who rely on it will be ignored then. like setting a puppet parameter to undef.

Reply:
but i'm no expert. but i'm having a hard time testing it. because i dont have a s3 setup and the one i have, i cannot access without making it a hassle. 🤔  :(